2-promises/solution.js

```diff
@@ -23,17 +23,29 @@ const id = yourRandomMethod() //third run
 6. use promise chaining accordingly, don't forget error handling
 7. log the resultant fullname, or the error, at the end
 */
+const firstNameMethodImport = require("./firstnames")
+const lastNameMethodImport = require("./lastnames")
+
+const ramdonMethodId = () => Math.floor(Math.random() * 101)
 
 function solution() {
     // YOUR SOLUTION GOES HERE
 
     // You generate your id value here
-
+    const idValue = ramdonMethodId();
```

Comment:
More descriptive variable names
```suggestion
    const id = ramdonMethodId();    
```